src/styles/tokens.scss

```diff
@@ -0,0 +1,187 @@
+@use 'sass:string';
```

Comment:
Overall, this file seems extremely complicated and could maybe use some simplification by splitting parts out if possible. These tokens are generated and shown to the user, so they should be a clean list of name/value pairs that people can quickly decipher. Look at any component's "Theme Support" section for a reference.

Additionally, the names of the tokens that can be overriden by users need to follow the pattern of `--ds-auro-tail-...`. Let me know if you have any more questions.